docs/guide/deployer.md

```diff
@@ -42,3 +45,29 @@ Smart contracts must be stored in `assets` folder.
 The main difference between deploying an ASA and SSC is that ASA takes `asset-name` and `ASADeploymentFlags` as input and SSC takes `smart-contract-names` and `SSCDeploymentFlags` as input.
 
 You can learn more about the flags from [Deployer API](https://scale-it.github.io/algo-builder/);
+
+#### Stateless Smart Contracts
+
+- *Contract Mode:*
+
+  Contract accounts act in a similar fashion to escrow accounts, where when the smart contract is compiled it produces an Algorand address. This address can accept Algos or Algorand ASAs with standard transactions, where the contract’s address is the receiver of the transaction.
+
+   Check our [examples/htlc-pyteal-ts](../examples/htlc-pyteal-ts) project to explore how to deploy Stateless Smart Contracts(lsig). In the file `scripts/deploy.ts`, you will find:
+
+  ```
+  await deployer.fundLsig('htlc.py',
+    { funder: bob, fundingMicroAlgo: 2e6 }, {}, [], scTmplParams);
+  ```
+  `fundLsig` funds the contract account (compiled hash of the smart contract). The function `fundLsig` accepts `pyteal` code too, which provides the functionality of dynamically providing the params before compiling into TEAL code.
+
+- *Delegated Signature*:
+
+  Stateless smart contracts can also be used to delegate signature authority. When used in this mode, the logic of the smart contract is signed by a specific account or multi-signature account. This signed logic can then be shared with another party that can use it to withdrawal Algos or Algorand ASAs from the signing account, based on the logic of the contract.
+
+  Use `mkDelegatedLsig` function to compile and sign a logic signature & save it to checkpoint.
+  ```javascript
+  const ascInfoGoldDelegated = await deployer.mkDelegatedLsig('4-gold-asa.teal', goldOwner, []);
+  console.log(ascInfoGoldDelegated);
+  ```
+
+You can learn more about Statless Smart Contracts [here](https://developer.algorand.org/docs/features/asc1/stateless/).
```

Comment:
```suggestion
You can learn more about Stateless Smart Contracts [here](https://developer.algorand.org/docs/features/asc1/stateless/).
```
how about adding this link at top of this subsection ?

Reply:
We can assume that most of the users will be aware about these docs, so that's why I put the links at the end. @ratik21 What do you say, should we move this to the top?

Reply:
I don't have a strong preference, but i think 'more info' links should be placed at the bottom of the subsection. OR we can just link the subsection heading with the more info link. @man-of-code your call.

Reply:
I prefer it this way.